doc/en/getistio-cli/reference/getistio_gen-ca/_index.md

```diff
@@ -9,6 +9,122 @@ Generates intermediate CA from different managed services such as AWS ACMPCA, GC
 getistio gen-ca [flags]
 ```
 
+#### Examples
+
+```
+- AWS:
+
+cat <<EOF >> aws.yaml
+providerName: aws-example
```

Comment:
can we change  "aws-example" to 'aws'. Provider name currently takes only two values 'aws' and 'gcp'.

There is an issue to remove this field and infer it from the providerConfig struct itself. We couldn't get to work on it.